src/gui/transferlistfilterswidget.cpp

```diff
@@ -561,7 +561,6 @@ QStringList TrackerFiltersList::getHashes(int row)
 TransferListFiltersWidget::TransferListFiltersWidget(QWidget *parent, TransferListWidget *transferList)
     : QFrame(parent)
     , m_transferList(transferList)
-    , m_trackerFilters(nullptr)
```

Comment:
And why is it superfluous?

Reply:
https://github.com/qbittorrent/qBittorrent/pull/9218/files/288a7fee50c3cf3c3192ba8dd581ffd867f2b6d2..f70f3fc7834aa823cda4a8af04131b9d50c9c2d9#diff-6f7b48e8ecc117c28a7f208770be0a96L642

Reply:
Yes, thanks; don't quite understand why do you replace assigments with initialisations in other cases, but do the opposite here?

Reply:
`frameLayout->addWidget(m_trackerFilters);` is on the next line, I'm not sure is it preferable to split them.